File: jphp-javafx-ext/src/main/resources/JPHP-INF/sdk/ext/javafx/layout/UXAnchorPane.php
<?php
namespace ext\javafx\layout;

use ext\javafx\layout\UXPane;
use ext\javafx\UXNode;

/**
 * Class UXAnchorPane
 * @package ext\javafx\layout
 */
class UXAnchorPane extends UXPane
{
    /**
     * @param UXNode $node
     * @return double
     */
    static function getLeftAnchor(UXNode $node) {}

    /**
     * @param UXNode $node
     * @return double
     */
    static function getRightAnchor(UXNode $node) {}

    /**
     * @param UXNode $node
     * @return double
     */
    static function getTopAnchor(UXNode $node) {}

    /**
     * @param UXNode $node
     * @return double
     */
    static function getBottomAnchor(UXNode $node) {}

    /**
     * @param UXNode $node
     * @param double $value
     */
    static function setLeftAnchor(UXNode $node, $value) {}

    /**
     * @param UXNode $node
     * @param double $value
     */
    static function setRightAnchor(UXNode $node, $value) {}

    /**
     * @param UXNode $node
     * @param double $value
     */
    static function setTopAnchor(UXNode $node, $value) {}

    /**
     * @param UXNode $node
     * @param double $value
     */
    static function setBottomAnchor(UXNode $node, $value) {}
}

File: exts/jphp-gui-ext/src/main/java/org/develnext/jphp/ext/javafx/classes/layout/UXAnchorPane.java
package org.develnext.jphp.ext.javafx.classes.layout;

import javafx.scene.Node;
import javafx.scene.layout.AnchorPane;
import org.develnext.jphp.ext.javafx.JavaFXExtension;
import php.runtime.annotation.Reflection.Name;
import php.runtime.annotation.Reflection.Signature;
import php.runtime.env.Environment;
import php.runtime.reflection.ClassEntity;

@Name(JavaFXExtension.NS + "layout\\UXAnchorPane")
public class UXAnchorPane<T extends AnchorPane> extends UXPane<AnchorPane> {
    interface WrappedInterface {

    }

    public UXAnchorPane(Environment env, T wrappedObject) {
        super(env, wrappedObject);
    }

    public UXAnchorPane(Environment env, ClassEntity clazz) {
        super(env, clazz);
    }

    @Signature
    public void __construct() {
        __wrappedObject = new AnchorPane();
    }

    @Signature
    public static void setLeftAnchor(Node node, double value) {
        AnchorPane.setLeftAnchor(node, value);
    }

    @Signature
    public static void setRightAnchor(Node node, double value) {
        AnchorPane.setRightAnchor(node, value);
    }

    @Signature
    public static void setTopAnchor(Node node, double value) {
        AnchorPane.setTopAnchor(node, value);
    }

    @Signature
    public static void setBottomAnchor(Node node, double value) {
        AnchorPane.setBottomAnchor(node, value);
    }

    @Signature
    public static double getLeftAnchor(Node node) {
        return AnchorPane.getLeftAnchor(node);
    }

    @Signature
    public static double getRightAnchor(Node node) {
        return AnchorPane.getRightAnchor(node);
    }

    @Signature
    public static double getTopAnchor(Node node) {
        return AnchorPane.getTopAnchor(node);
    }

    @Signature
    public static double getBottomAnchor(Node node) {
        return AnchorPane.getBottomAnchor(node);
    }

    @Signature
    public static void setAnchor(Node node, double value) {
        AnchorPane.setBottomAnchor(node, value);
        AnchorPane.setTopAnchor(node, value);
        AnchorPane.setRightAnchor(node, value);
        AnchorPane.setLeftAnchor(node, value);
    }
}
